discord/ext/commands/help.py

```diff
@@ -570,12 +571,14 @@ async def filter_commands(self, commands, *, sort=False, key=None, exclude: tupl
             key = lambda c: c.name
 
         # Ignore Application Commands because they don't have hidden/docs
+        from ...commands import ApplicationCommand  # noqa: PLC0415
```

Comment:
why?

Reply:
I think it broke when I changed the import for the utils stuff. I will change it not to have  the ``...commands` however.

<img width="1206" height="174" alt="Image" src="https://github.com/user-attachments/assets/7cfbe594-03a3-49cb-9d8d-9059dced1640" />